Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -998,16 +997,15 @@ class BrowserViewController: UIViewController {
         urlBar.tabLocationViewDidTapLocation(urlBar.locationView)
     }
 
-    private func popToTab(forTab: Tab? = nil) {
+    private func popToTab() {
         guard let currentViewController = navigationController?.topViewController else {
                 return
         }
         if let presentedViewController = currentViewController.presentedViewController {
             presentedViewController.dismissViewControllerAnimated(false, completion: nil)
         }
         // if a tab already exists and the top VC is not the BVC then pop the top VC, otherwise don't.
-        if currentViewController != self,
-            let _ = forTab {
+        if currentViewController != self {
             self.navigationController?.popViewControllerAnimated(true)
```

Comment:
Does this fix the bug?

The exit state of this method is that BVC should be top most. However, this assumes that it's the one underneath the current top most. Is this always true?